packages/logs/src/domain/logger/loggerCollection.ts

```diff
@@ -51,3 +58,11 @@ export function isAuthorized(status: StatusType, handlerType: HandlerType, logge
     STATUS_PRIORITIES[status] >= STATUS_PRIORITIES[logger.getLevel()] && includes(sanitizedHandlerType, handlerType)
   )
 }
+
+function displayInConsole(logsMessage: LogsMessage, messageContext: Context | undefined) {
+  const consoleMethod = Object.prototype.hasOwnProperty.call(originalConsoleMethods, logsMessage.status)
+    ? originalConsoleMethods[logsMessage.status]
+    : originalConsoleMethods.log
+
+  consoleMethod.call(globalConsole, logsMessage.message, messageContext)
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding some tests for:
- ensuring that there is no prefix in this case
- console.log is used when the status is not in originalConsoleMethods

Reply:
> **💬 suggestion:** ‏what about adding some tests for:
> 
>     * ensuring that there is no prefix in this case

Added a E2E test for it

> 
>     * console.log is used when the status is not in originalConsoleMethods

I removed this logic, because it cannot happen in practice: all `StatusType` has a matching `ConsoleApiName`, and `isAuthorized` makes sure the logs `status` is valid. [This test](https://github.com/DataDog/browser-sdk/blob/2d4e4caa73fc01970c85dddcadf647fc3e3c084e/packages/logs/src/domain/logger/loggerCollection.spec.ts#L100) makes sure things do not break when using an unknown status type